dexter/README.md

```diff
@@ -0,0 +1,175 @@
+# Dexter
+
+Dexter is a logging middleware inspired by [expressjs/morgan](https://github.com/expressjs/morgan). It is configurable and can be used throughout the request-resource-response lifecycle.
+
+```typescript
+import { Drash } from "https://deno.land/x/drash@{version}/mod.ts";
+
+// Import the Dexter middleware function
+import { Dexter } from "https://deno.land/x/drash-middleware@{version}/dexter/mod.ts";
+
+// Instantiate dexter
+const dexter = Dexter();
+
+// Create your server and plug in dexter to the middleware config
+const server = new Drash.Http.Server({
+  resources: [
+    HomeResource,
+  ],
+  middleware: {
+    before_request: [
+      dexter
+    ],
+    after_request: [
+      dexter
+    ]
+  }
+});
+
+server.run({
+  hostname: "localhost",
+  port: 1447,
+});
+
+console.log(`Server running at ${server.hostname}:${server.port}`);
+```
+
+## Configuration
+
+If you decide to configure Dexter, make sure you specify the `enabled` flag in the configs as it is required when customizing the configuration.
+
+### `enabled`
+
+Enable or disable the logger from logging based on the value of this config.
+
+```typescript
+const dexter = Dexter({
+  enabled: true, // or false
+});
+```
+
+### `level`
+
+Define what log statements should be written based on their log level definition (e.g., debug, info, warn).
+
+```typescript
+const dexter = Dexter({
+  enabled: true,
+  level: "debug", // or all, trace, debug, info, warn, error, fatal, off
```

Comment:
what's the input for multiple levels?  "trace,info",  ["trace", "info"], or... ?

Reply:
you can't have multiple levels